operator-pipeline-images/operatorcert/entrypoints/hydra_checklist.py

```diff
@@ -74,7 +81,7 @@ def main() -> None:
         log_level = "DEBUG"
     logging.basicConfig(level=log_level)
 
-    check_hydra_checklist_status(args.cert_project_id, args.hydra_url)
+    check_hydra_checklist_status(args.cert_project_id, args.hydra_url,args.developer)
```

Comment:
Hey! Current solution looks nice but it won't work as intended, as the `args.developer` passed from the pipeline to script is a string and not bool- so this step will always be skipped. Please, next time test as much of variants as possible for your features. 
To make things simpler, maybe we could just not trigger `hydra-checklist` script in case flag is set to true- the same way as it happens [here](https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/ansible/roles/operator-pipeline/templates/openshift/tasks/preflight-trigger.yml#L42)?

Reply:
Yes, I know this fact that with the current setup the code won't work. I just added it to ask the team for suggestions, the PR is still in draft. And according to the [AC], the hydra-checklist should be triggered even after the flag, the task is to prevent the failure while the developer is running the pipeline for testing purposes. WDYT?

Reply:
We should still query the publishing checklist because there is value in testing its availability and correctness of the response. For testing purposes though, it's nice to be able to ignore the `completed` state.

Reply:
I though, that we want to skip the task completelly- if we still want to query Hydra, I agree with Alex.